config/service_script.conf

```diff
@@ -0,0 +1,21 @@
+[supervisord]  ## This is the main process for the Supervisor
+nodaemon=true  ## This setting is to specify that we are not running in daemon mode
+
+## agora_service
+[program:agora_service]
+command=agora -c config/agora_config.yaml
```

Comment:
Does this work ? What WD is used ?

Reply:
What is WD?

Reply:
Working directory

Reply:
Yes, it works. 
It's here. /usr/local/bin/agora